src/deploy/osp_deployer/director.py

```diff
@@ -1723,15 +1736,36 @@ def configure_dashboard(self):
         logger.info("Configure Dashboard")
         ip = self.settings.dashboard_node.public_api_ip
 
-        self.run_tty(self.source_stackrc + 'cd ' +
-                     self.pilot_dir +
-                     ';./config_dashboard.py ' +
-                     ip +
-                     ' ' + self.settings.dashboard_node.root_password +
-                     ' ' + self.settings.subscription_manager_user +
-                     ' ' + self.settings.subscription_manager_password +
-                     ' ' + self.settings.subscription_manager_pool_sah +
-                     ' ' + self.settings.subscription_manager_vm_ceph)
+        if self.settings.use_satellite is True:
+            self.run_tty(self.source_stackrc + 'cd ' +
+                         self.pilot_dir +
+                         ';./config_dashboard.py --dashboard_addr ' +
```

Comment:
A NIT and FYI,  I rather see --dashboard_addr with it's IP on same line ... a tiny bit clearer to read.

Reply:
I beleive the reason here is it would push the length of the line over the pep compliance limit.

Reply:
Not if you move --dashboard_addr down to same line as ip.   Not a big deal either way,  just commenting so that you know I looked at your changes.

Reply:
done